src/components/Requirements/CompletedSubReqCourse.vue

```diff
@@ -73,22 +68,16 @@ export default Vue.extend({
         return { semesterLabel: 'Transfer Credits', color: transferCreditColor, uniqueID: 0 };
```

Comment:
Should `uniqueID` be -1 since `this.courseTaken.uniqueId === -1` for transfer credits? What's the distinction between `uniqueId` and `uniqueID`?